.github/workflows/docs.yml

```diff
@@ -0,0 +1,48 @@
+name: Reusable workflow that builds documentation locally
+
+on:
+  workflow_call:
+
+permissions:
+  contents: read
+
+env:
+  # Environment variables to support color support (jaraco/skeleton#66):
+  # Request colored output from CLI tools supporting it. Different tools
+  # interpret the value differently. For some, just being set is sufficient.
+  # For others, it must be a non-zero integer. For yet others, being set
+  # to a non-empty value is sufficient. For tox, it must be one of
+  # <blank>, 0, 1, false, no, off, on, true, yes. The only enabling value
+  # in common is "1".
+  FORCE_COLOR: 1
```

Comment:
As one fellow in Brisbane WICEN puts it… there are 3 certainties in life:

1. death
2. taxes
3. what F##$!ing standards

:-)